scripts/Engineering/texture/TextureUtils/multidomain_engine.py

```
@@ -0,0 +1,524 @@
+        # convert d spacing to tof - better step size for A, B and S refinement
+        diff_consts = si.diffractometerConstants(ispec)
+        tof_peak = _d_to_tof(peak, diff_consts)
+        tof_start = _d_to_tof(x_start, diff_consts)
+        tof_end = _d_to_tof(x_end, diff_consts)
+
+        # get the window indices for the spectra
+        istart = ws_tof.yIndexOfX(tof_start, ispec)
+        iend = ws_tof.yIndexOfX(tof_end, ispec)
+
+        # get param estimates
+        intens, sigma, bg, centre = _estimate_intensity_background_and_centre(ws_tof, ispec, istart, iend, tof_peak)
```

Ruff (0.15.20)

[warning] Unpacked variable `sigma` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)